ui/src/profiles/EditProfile/EditProfile.tsx

```diff
@@ -214,17 +218,58 @@ function EditProfileContent() {
   );
 }
 
+const pageAnimationVariants = {
+  initial: {
+    opacity: 0,
+    x: '100vw',
+  },
+  in: {
+    opacity: 1,
+    x: 0,
+  },
+  out: {
+    opacity: 0,
+    x: '-100vw',
+  },
+};
+
+const pageTransition = {
+  type: 'tween',
+  ease: 'easeInOut',
+  duration: 0.2,
+};
+
 export default function EditProfile({ title }: ViewProps) {
   useAnalyticsEvent('profile_edit');
+  const isMobile = useIsMobile();
 
   return (
     <div className="flex grow overflow-y-scroll bg-gray-50">
       <Helmet>
         <title>{title}</title>
       </Helmet>
-      <div className="w-full p-6">
-        <EditProfileContent />
-      </div>
+      <Layout
+        header={
+          isMobile ? (
+            <MobileHeader title="Edit Profile" pathBack="/profile" />
```

Comment:
this leaves me stranded if clicked from Talk

Reply:
How did you get to this view on mobile talk?

Reply:
Grid icon in the header → Profile

Reply:
Ah, via leap, got it.